controllers/placementapi_controller.go

```diff
@@ -967,6 +992,40 @@ func (r *PlacementAPIReconciler) findObjectsForSrc(ctx context.Context, src clie
 	return requests
 }
 
+func (r *PlacementAPIReconciler) findObjectsForNamespace(ctx context.Context, src client.Object) []reconcile.Request {
+	requests := []reconcile.Request{}
+
+	l := log.FromContext(context.Background()).WithName("Controllers").WithName("PlacementAPI")
+
+	// if the endpoint has the service label and its in our endpointList, reconcile the CR in the namespace
+	if svc, ok := src.GetLabels()[common.AppSelector]; ok && util.StringInSlice(svc, endpointList) {
```

Comment:
only reconcile if the service has the expected label and one of the names in our endpointList